lib/texts/TextOperations.cpp

```diff
@@ -300,30 +301,62 @@ int TextOperations::getLevenshteinDistance(const std::string & s, const std::str
 	return v0[n];
 }
 
-bool TextOperations::textSearchSimilar(const std::string & s, const std::string & t)
+DLL_LINKAGE std::string TextOperations::getLocaleName()
+{
+	try
+	{
+		const std::string localeName = Languages::getLanguageOptions(LIBRARY->generaltexth->getPreferredLanguage()).localeName;
+
+		if(localeName.empty())
+			throw std::runtime_error("Using fallback: en_US.UTF-8");
+
+		return localeName;
+	}
+	catch(const std::exception & e)
+	{
+		logGlobal->warn("Failed to retrieve locale. %s", e.what());
+	}
+
+	return "en_US.UTF-8";
+}
```

Comment:
1. Why you need exception throwing? Just warning + return would work just as well
2. In which conditions this might happen? From what I understand, it can never happen.

I think this entire method can be replaced with simple 1-liner
```suggestion
DLL_LINKAGE std::string TextOperations::getLocaleName()
{
	return Languages::getLanguageOptions(LIBRARY->generaltexth->getPreferredLanguage()).localeName;
}
```

Reply:
Totally right. It's leftover when I was trying to set up `std::locale` here. Should it be moved to header and marked with `inline` ?

Reply:
Can be kept in .cpp

Or alternatively - remove entirely (if it is only used in 1 place)